aws-go-lambda/README.md

```diff
@@ -0,0 +1,61 @@
+# AWS Golang Lambda
+This example creates a lambda that does a simple `ToUpper` on the string input and returns it.
+
+## Deploying the App
+
+ To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+2. [Configure AWS Credentials](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/)
+3. [Clone aws-go-lambda](https://github.com/aws/aws-lambda-go)
+
+### Steps
+
+After cloning this repo, run these commands from the working directory:
+
+1. Build the go files:
+
+```bash
+make build
```

Comment:
Btw - I’m hopeful that once we fix https://github.com/pulumi/pulumi/issues/3593 we can automate this directly within the Pulumi program so that the separate step is not needed - but today I think it is so I like having this example as is until then.